baal/active/heuristics/heuristics.py

```diff
@@ -12,6 +12,13 @@
 
 from baal.utils.array_utils import to_prob
 
+DEPRECATED = "DEPRECATED"
+SHUFFLE_PROP_DEPRECATION_NOTICE = """
+`shuffle_prop` is deprecated and will be removed in Baal 1.9.0.
+To upgrade, use `baal.active.heuristics.stochastics.PowerSampling`.
```

Comment:
```suggestion
For better batch uncertainty estimation, use `baal.active.heuristics.stochastics.PowerSampling`.
```

Reply:
I feel `upgrade` takes my mind to upgrading the repo instead of better method to use